packages/main/src/TextArea.js

```diff
@@ -570,14 +570,15 @@ class TextArea extends UI5Element {
 
 	get styles() {
 		const lineHeight = 1.4 * 16;
+		const mainHeight = (this.rows * lineHeight) + (this.showExceededText ? 32 : 0);
```

Comment:
Overall the change looks good, but I have a single question. How do you decide exactly 32, where it comes from?

Reply:
This is a "magic number" like the ones above :) 
Basically, this is the height of the line showing exceeded characters. It was never taken into account.

Reply:
Per my understanding, the necessary height for the exceeded text is the padding applied + the font size => 26px is enough. Could you please check?